assignments/callbacks.js

```diff
@@ -1,50 +1,64 @@
-// Create a higher order function and invoke the callback function to test your work. You have been provided an example of a problem and a solution to see how this works with our items array.  Study both the problem and the solution to figure out the rest of the problems.
+// Create a higher order function and invoke the callback function to test your work. 
+//You have been provided an example of a problem and a solution to see how this works with our items array.  
+//Study both the problem and the solution to figure out the rest of the problems.
 
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
-/* 
+
 
   //Given this problem: 
   
-  function firstItem(arr, cb) {
+  // function firstItem(arr, cb) {
     // firstItem passes the first item of the given array to the callback function.
-  }
+  // }
 
   // Potential Solution:
 
   // Higher order function using "cb" as the call back
-  function firstItem(arr, cb) {
-    return cb(arr[0]);
-  }
+  // function firstItem(arr, cb) {
+    // return cb(arr[0]);
+  // }
 
   // Function invocation 
-  firstItem(items, function(first) {
-    console.log(first)
-  });
+  // firstItem(items, function(first) {
+    // console.log(first)
+  // });
+
 
-*/
 
 
-function getLength(arr, cb) {
+function getLength(items, cb) {
   // getLength passes the length of the array into the callback.
-}
+  return cb(items.length);
+};
+console.log(getLength(items, length => items.length))
```

Comment:
Much of the function is written in the console.log and the name for the 
```suggestion
console.log(getLength(items, ( ) => items.length))
```